.changeset/tiny-donkeys-suffer.md

```diff
@@ -0,0 +1,13 @@
+---
+'skeleton': patch
+---
+
+Since Remix is now a peer dependency of `@shopify/remix-oxygen`, you need to add `@remix-run/server-runtime` to your dependencies with the same version you have for the rest of Remix dependencies.
+
+```diff
+"dependencies": {
+  "@remix-run/react": "2.1.0"
++ "@remix-run/server-runtime": "2.1.0"
+  ...
+}
+```
```

Comment:
`@remix-run/server-runtime` was marked as a peer dependency in https://github.com/Shopify/hydrogen/pull/1484. This is not what [other Remix adapters](https://github.com/remix-run/remix/blob/main/packages/remix-cloudflare/package.json#L16) do but I agree it makes sense for us since we don't follow the same versioning as other adapters.
The downside is that users should install `@remix-run/server-runtime` as well as the adapter.

Reply:
Yes this is actually a "bug fix". Luckily it doesn't even break anything because `@remix-run/server-runtime` is already installed by other Remix packages I think.